src/python/pants/rules/core/fastlist.py

```diff
@@ -15,3 +16,15 @@ def fast_list(console, addresses):
   """A fast variant of `./pants list` with a reduced feature set."""
   for address in addresses.dependencies:
     console.print_stdout(address.spec)
+
+
+# This should really be registered somehow as an in-repo plugin when testing, rather than living
```

Comment:
Making this an explicit TODO would be good.

Reply:
Doesn't TestBase support this via:
https://github.com/pantsbuild/pants/blob/17ddd3938e09b6db0b5fc9561efb3c53f8501699/tests/python/pants_test/test_base.py#L275

Or do console rules need special setup that TestBase doesn't handle? A naive scan through the relevant code looks like it should all work.

Reply:
Moved into pants-plugins/src/python/internal_backend/rules_for_testing